roles/splunk/tasks/configure_dmc.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
```suggestion
---
- name: Check splunk
  include_task: check_splunk.yml

- name: Configure DMC as a IDXC member
  include_tasks: configure_idxc_sh.yml
```

Reply:
Change integrated in [34b733a](https://github.com/splunk/ansible-role-for-splunk/pull/125/commits/34b733a9cfddcaafb0c3e7348aa58d00b9053fd3)